src/ResourceManager/Profile/Commands.Profile/AzureRmAlias/Mappings.json

```diff
@@ -2281,6 +2281,9 @@
     "Get-AzStorageAccountNetworkRuleSet": "Get-AzureRmStorageAccountNetworkRuleSet",
     "Add-AzStorageAccountNetworkRule": "Add-AzureRmStorageAccountNetworkRule",
     "Remove-AzStorageAccountNetworkRule": "Remove-AzureRmStorageAccountNetworkRule",
+    "Set-AzStorageAccountManagementPolicy": "Set-AzureRmStorageAccountManagementPolicy",
```

Comment:
There is no need for this.  These cmdlet aliases will exist by default.

Reply:
the change to mappings.json is reverted.